style.css

```diff
@@ -227,251 +242,352 @@ li:hover {
+  background-position: 180px 20px;
+  margin-top: -98px;
+  visibility: visible;
+  background-size: 200px;
+  border-top-left-radius: 50px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  flex-grow: 0;
+  flex-wrap: nowrap;
+  justify-items: center;
 }
 
-#fra {
-  height: 24px;
-  left: 12px;
-  right: 78px;
+#Me {
+  align-items: center;
   font-family: 'Poppins', 'Times New Roman', Times, serif;
   font-style: normal;
-  font-weight: 500;
-  font-size: 20px;
-  line-height: 24px;
-  display: flex;
-  align-items: center;
-  color: #000;
-  padding-left: 30px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  width: 100%;
+  color: #fff;
+  padding-top: 80px;
+  margin-top: 15px;
+  margin-right: 51px;
+  margin-bottom: 51px;
+  margin-left: 51px;
 }
 
-#sk {
-  left: 12px;
-  right: 78px;
+.Contact-me p {
+  order: 1;
+  margin-top: 61px;
   font-family: 'Poppins', 'Times New Roman', Times, serif;
   font-style: normal;
-  font-weight: 500;
+  font-weight: 400;
   font-size: 20px;
-  line-height: 24px;
-  align-items: center;
-  color: #000;
-  padding-left: 30px;
-  display: inline-block;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
 }
 
-.fa-angle-right {
-  padding-left: 280px;
+.Click-1 {
+  margin-top: 450px;
+  float: right;
+  order: 2;
+  flex-direction: column;
+  align-items: center;
 }
 
-.il {
-  display: inline;
-  padding: 10px;
+.Details {
+  height: 38px;
+  width: 316px;
+  border-radius: 8px;
+  border: #dfe1e6;
+  margin-left: 24px;
+  margin-bottom: 18px;
 }
 
-.fs {
-  display: inline;
-  width: 100px;
-  padding-left: -20px;
+#Details {
+  height: 170px;
+  width: 316px;
+  border-radius: 8px;
+  border: #dfe1e6;
+  margin-left: 24px;
 }
 
-.fa-angle-down {
-  padding-left: 110px;
-} */
-
-#la {
-  margin-top: 30px;
-  margin-bottom: 30px;
-  height: 24px;
+.button-3 {
+  width: 105px;
+  height: 35px;
   font-family: 'Poppins', 'Times New Roman', Times, serif;
   font-style: normal;
   font-weight: 500;
-  font-size: 20px;
-  line-height: 24px;
-  align-items: center;
-  color: #000;
-  display: inline-flex;
-  padding-left: 30px;
+  font-size: 12px;
+  background-color: white;
+  color: #396df2;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  margin-left: 23px;
+  cursor: pointer;
+  margin-top: 10px;
 }
 
-.box :hover {
-  transform: scale(1.5);
-  cursor: pointer;
+::placeholder {
+  font-family: 'Poppins', 'Times New Roman', Times, serif;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 20px;
+  letter-spacing: 0;
+  text-indent: 14px;
+  position: absolute;
+  margin-top: 10px;
 }
 
-.pics img {
-  width: 303px;
-  left: 12px;
-  border-radius: 8px;
-  padding-left: 30px;
+button:enabled {
+  background-color: white;
 }
 
-.skills {
-  display: inline-block;
-  flex-direction: column;
-  flex-wrap: wrap;
-  justify-content: center;
-  align-items: center;
+button:hover {
```

Comment:
- [ ] As suggested by the previous reviewer, please add the correct background color on hover, , active (when it's pressed).

![Screenshot from 2022-04-27 19-12-50](https://user-images.githubusercontent.com/52934128/165592912-7d8bf224-8e6b-467d-bb19-cf63cd2b40fd.png)

the button on hover has the `background: #6070FF;` and `color: #FFFFFF`, 

**for hover state**

```
button:hover {
    background-color: #6070FF;
    color: #FFFFFF;
}
```